app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+  has_many :posts
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(1)
```

Comment:
- [ ] Kindly consider fixing this line so that it will return 3 most recent posts to meet the below requirement 👇 
> Users
A method that returns the 3 most recent posts for a given user.